tests/src/JIT/CodeGenBringUpTests/Rotate.cs

```diff
@@ -128,12 +146,30 @@ static ulong ror64comm(ulong value, int amount)
     [MethodImpl(MethodImplOptions.NoInlining)]
     static ulong ror64const()
     {
-        ulong value = flag() ? (ulong)0x123456789abcdef : (ulong)0x123456789abcdef;
+        ulong value = flag() ? (ulong)0x123456789abcdef : (ulong)0xabcdef123456789;
         int amount = flag() ? 5 : 5;
```

Comment:
It seems like the line below:
```
int amount = flag() ? 5 : 5;
```
is intended to make the rotate by a variable amount? Don't we optimize that away, though, and then constant fold the shift amounts? Or, the name "ror64const" I guess implies the rotation amount should be constant, then why do we need to `?:`?